### PR TITLE
Is 12465

### PR DESCRIPTION
**Pull request by erik-leven**
Skjermdump fra 2022-05-11 09-11-03
Skjermdump fra 2022-05-11 09-10-44

### CODE REVIEW

File: tutorial-getting-started-enrich.rst [hunk 1]
```diff
@@ -1,32 +1,124 @@
...
+When adding semantic value to our data we create a more scalable solution. We also create a more efficient solution as the enrich data both allows us to visualize the relationships in the source data as well as increases functionality throughout the whole data flow. 
+
+The Hubspot Data
+****************
+In order to add the link between associated contacts for a company and contacts, as well as adding the rdf:type tag to the entities, please follow the steps below:
+
+#. Navigate to **Pipes**
+#. Click on **New pipe**
+#. Paste and save the DTL configuration below
+#. Press **Start** to ensure your pipe runs
```
Comment (ga-hegsvold) [inline tutorial-getting-started-enrich.rst:40]: "Press" --> "Click"?

File: tutorial-getting-started-enrich.rst [hunk 2]
```diff
@@ -1,32 +1,124 @@
...
+
+The Hubspot Data
+****************
+In order to add the link between associated contacts for a company and contacts, as well as adding the rdf:type tag to the entities, please follow the steps below:
+
+#. Navigate to **Pipes**
+#. Click on **New pipe**
+#. Paste and save the DTL configuration below
+#. Press **Start** to ensure your pipe runs 
+#. Press refresh to see number of entities processed (should be 10). You can also see them in the pipe's output page.
```
Comment (ga-hegsvold) [inline tutorial-getting-started-enrich.rst:41]: "Press" --> "Click"?

File: tutorial-getting-started-enrich.rst [hunk 3]
```diff
@@ -1,32 +1,124 @@
...
+On the output entity ``hubspot-company:5584839113`` you should now see a new property called ``contact-ni`` which contains the link to contacts, as well as namespaces on every property and the new ``rdf:type`` property.
+
+The Enhetsregisteret Data
+*************************
+For the Enhetsregisteret data we will only add namespaces and the ``rdf:type`` property. Follow the steps below to create the Enrich pipe for the Enhetsregisteret data.
+
+#. Navigate to **Pipes**
+#. Click on **New pipe**
+#. Paste and save the DTL configuration below
+#. Press **Start** to ensure your pipe runs
```
Comment (ga-hegsvold) [inline tutorial-getting-started-enrich.rst:92]: "Press" --> "Click"?

File: tutorial-getting-started-enrich.rst [hunk 4]
```diff
@@ -1,32 +1,124 @@
...
+
+The Enhetsregisteret Data
+*************************
+For the Enhetsregisteret data we will only add namespaces and the ``rdf:type`` property. Follow the steps below to create the Enrich pipe for the Enhetsregisteret data.
+
+#. Navigate to **Pipes**
+#. Click on **New pipe**
+#. Paste and save the DTL configuration below
+#. Press **Start** to ensure your pipe runs 
+#. Press refresh to see number of entities processed (should be 10). You can also see them in the pipe's output page.
```
Comment (ga-hegsvold) [inline tutorial-getting-started-enrich.rst:93]: "Press" --> "Click"?

File: tutorial-getting-started-enrich.rst [hunk 5]
```diff
@@ -1,32 +1,124 @@
...
+      "transform": {
+        "type": "dtl",
+        "rules": {
+          "default": [
+            ["copy", "*"],
+            ["merge",
+              ["apply", "contacts-ni", "_S.associations.contacts.results"]
+            ],
+            ["add", "rdf:type",
+              ["ni", "hubspot:company"]
```
Comment (ga-hegsvold) [inline tutorial-getting-started-enrich.rst:62]: I guess it's not really an "official" naming convention, but in most projects I've work on we've used capital letter on the rdf:type.

Example: "hubspot:Company".

I think this makes it clearer that we're defining the entity type/business object here.
Thoughts?

File: tutorial-getting-started-enrich.rst [hunk 6]
```diff
@@ -1,32 +1,124 @@
...
-
 .. _tutorial_getting_started_enrich:
 
 Enrich data
 ===========
 
-In this tutorial we will look closer into how to...
+In this tutorial we will look closer into how to semantically enrich your data. We add semantic value to our data in Sesam by identifying reference objects in the data and link them to their associated target, such as the link between a Primary Key (PK) and a Foreign Key (FK). In Sesam we use namespaced identifiers (NI's) and rdf:types to create these links. We also use the semantic enrichment step to add namespaces to our data. 
+
+In this tutorial we will only add some enrichment, in order for you to understand the concept. To semantically enrich data you need to know what the data means, which is why semantic enrichment should always be done with someone with more intimate knowledge about the data, such as a system owner.
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:8]: Yeah, I like it

File: tutorial-getting-started-enrich.rst [hunk 7]
```diff
@@ -1,32 +1,124 @@
...
 .. admonition::  Objectives:
-   
-    After you complete this tutorial you would have learned the following:
 
-    #. Objective
-    #. Objective
-    #. Objective
+    After you complete this tutorial you will have learned the following:
 
+    #. How add NI's to your entity metadata
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:14]: fixed

File: tutorial-getting-started-enrich.rst [hunk 8]
```diff
@@ -1,32 +1,124 @@
...
+    #. How add rdf:types to your entities
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:15]: fixed

File: tutorial-getting-started-enrich.rst [hunk 9]
```diff
@@ -1,32 +1,124 @@
...
-    - 
-    - 
-    
-  You should also acquire:
-    - 
-    - 
-    - 
-    - 
+  Before starting on this tutorial we suggest you:
+    - Complete the `Collect data tutorial <tutorial-getting-started-collect>`_ as we will use that data in this tutorial.
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:20]: agreed

File: tutorial-getting-started-enrich.rst (hunk 5 above)
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:62]: This was copied from the demo node. I'm not sure what the convention here is tbh. First glance, I think I prefer small letters

File: tutorial-getting-started-enrich.rst [hunk 10]
```diff
@@ -1,32 +1,124 @@
...
+
+  In order to simplify matters, we would like you to assign a contact to one of the companies you now have populated your HubSpot account with. To do this, go to your "Companies" overwiew in HubSpot. Select ``THEMOON AS``. On the right hand side you should see the option to add a contact. Press ``+ Add`` and select **Brian Halligan** (a pre-existing test contact) and finish the contact selection. **Brian Halligan** should now be visible as a contact for ``THEMOON AS``.
```
[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:22]: Press is used in the previous tutorial. (https://blogs.windows.com/windowsexperience/2009/07/14/press-click-select-or-choose/)

File: tutorial-getting-started-enrich.rst (hunk 1 above)
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:40]: above

File: tutorial-getting-started-enrich.rst (hunk 2 above)
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:41]: above

File: tutorial-getting-started-enrich.rst (hunk 3 above)
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:92]: above

File: tutorial-getting-started-enrich.rst (hunk 4 above)
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:93]: above

File: tutorial-getting-started-enrich.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:22]: We can bring it up on the Monday meeting, but push this through for now?

**Review by ga-hegsvold — APPROVED** [review]

File: tutorial-getting-started-enrich.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (ga-hegsvold) [reply tutorial-getting-started-enrich.rst:21]: I like the distinction made on the site you linked to:
_The guideline is to use “Press” when referring to a hardware button and “Click” when referring to an item in the UI (e.g. soft keys, list items, push buttons etc.) Don’t use “Select”._

[the remark this answers is not in the document]
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst:21]: Yeah, it was very concise

File: tutorial-getting-started-enrich.rst [hunk 11]
```diff
@@ -3,9 +3,7 @@
...
-
-In this tutorial we will not go through the full extent of semantic enrichment, only enough for you to understand the concept. To semantically enrich data you need to know what the data means, which is why semantic enrichment should always be done with someone with more intimate knowledge about the data, such as a system owner. 
+In this tutorial we will look closer into the Enrich phase of a Sesam synchronization, where we add value to our data through semantically enrichment. We will not go through the full extent of semantic enrichment, only enough for you to understand the concept. To semantically enrich data you need to know what the data means, which is why this phase should always be done with someone with more intimate knowledge about the data, such as a system owner.
```
Comment (ga-hegsvold) [inline tutorial-getting-started-enrich.rst:6]: I like the shorter into.
"semantically enrichment" should be "semantic enrichment", no?

**Review by ga-hegsvold — APPROVED** [review]

**Comment by ga-hegsvold** [comment]
We should also try to be consistent in how we write headings regarding capitalization. Currently there's a bit of a mix between upper and lower casing of words in the headings. I suggest we only capitalize the first word of a heading and keep the rest lower case as if writing a sentence (special terms normally capitalized being the exception).

File: tutorial-getting-started-enrich.rst (hunk 11 above)
Comment (erik-leven) [reply tutorial-getting-started-enrich.rst]: agreed, typo.. fixing now

**Comment by erik-leven** [comment]
> 

https://www.grammarly.com/blog/capitalization-in-the-titles/ Looks like there probably should be a mix in there.

**Comment by ga-hegsvold** [comment]
> https://www.grammarly.com/blog/capitalization-in-the-titles/ Looks like there probably should be a mix in there.

Looks like those are guidelines for book- and article titles primarily, maybe not for section headings?

Here's Wikipedia's approach to styling: https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Capital_letters
_Wikipedia avoids unnecessary capitalization._

Regarding headings, see "Headings, headers, and captions".

**Review by djalil-bouhachi — APPROVED** [review]

**Pull request merged**